convert2rhel/unit_tests/breadcrumbs_test.py

```diff
@@ -338,3 +342,28 @@ def test_set_target_os(pretend_os):
         "name": "CentOS Linux",
         "version": "7.9",
     } == breadcrumbs.breadcrumbs.target_os
+
+
+@pytest.mark.parametrize(
+    ("argv", "expected", "message"),
+    (
+        (
+            ["analyze"],
+            False,
+            "Convert2RHEL modifies the systems during the analysis and then rolls back these "
+            "changes when the analysis is complete. In rare cases, this rollback can fail. "
+            "By continuing, you confirm that you have made a system backup and verified that "
+            "you can restore from the backup.",
+        ),
+    ),
+)
+def test_confirm_user_backup(argv, expected, message, monkeypatch, global_tool_opts, caplog):
+    monkeypatch.setattr(sys, "argv", mock_cli_arguments(argv))
+    try:
+        convert2rhel.toolopts.CLI()
+    except SystemExit:
+        pass
```

Comment:
ToolOpts.CLI does not need to be initialized here. You could rather just test that the function you created is giving the correct warning output.

Reply:
Ahhh ok, Yeah I wasn't sure exactly how I was supposed to test this with it being for the analysis so I just copied one of the other test functions that I thought was close enough to what I needed! Thank you I will just test the function and make sure that It does display the warning!